homeassistant/components/octoprint/sensor.py

```diff
@@ -88,9 +96,9 @@ def name(self):
 class OctoPrintStatusSensor(OctoPrintSensorBase):
     """Representation of an OctoPrint sensor."""
 
-    def __init__(self, coordinator: DataUpdateCoordinator, sensor_name: str):
+    def __init__(self, coordinator: DataUpdateCoordinator, device_id: str):
```

Comment:
If the device id is the config entry entry_id, let's keep calling it `config_entry_id` or similar.

Reply:
Done